var/spack/repos/builtin/packages/blaspp/package.py

```diff
@@ -0,0 +1,81 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+# 1) The CMake options exposed by `blaspp` allow for a value called `auto`. The
+#    value is not needed here as the choice of dependency in the spec
+#    determines the appropriate flags.
```

Comment:
This is quite often the case, does not need to be documented here :)

Reply:
Would it be OK to leave the comment as it is? I will likely forget about the "auto" option a few months down the line, I'd rather have the note wait for me here.

Reply:
I just checked the upstream `CMakeLists.txt`: it seams they are `auto` by default which is good to overwrite as you did.
Do you maybe want to move the CMake-argument comment down to (or into) the `cmake_args` function, so it's closer to the args it describes?